static/src/js/components/ProjectCollections/ProjectCollectionItem.scss

```
@@ -68,7 +68,7 @@
 
   &__stats-item {
     margin-right: calc($spacer / 2);
-    color: $color__black--500;
+    color: $color__carbon--60;
```

```suggestion
    color: $color__carbon--50;
```
should this be 50 following the pattern

---

They don't match up so good in some places, so if they are different that is deliberate.  Probably for contrast requirements.